excel-connector-demo/src_hd/com/axonivy/connector/excel/demo/PersonEditor/PersonEditorProcess.mod

```diff
@@ -69,6 +69,9 @@ Ps0 f6 actionTable 'out=in;
 ' #txt
 Ps0 f6 actionCode 'import com.axonivy.connector.excel.demo.Person;
 
+out.recordsetKeys.add("firstname");
```

Comment:
instead of hardcoding these values, I think we should be able to traverse not only 'Person' but any Data object:
1. Person is a CompositeObject 
2. We can read it's field via this API: ch.ivyteam.ivy.scripting.objects.CompositeHelper.getFieldNames(CompositeObject)
3. CompositeObject.get(theFieldName) allows generic access without knowing any particular Data object.

I guess that the best place to do this conversion is in the connector itself:
param = List<CompositeObject> 
-- transform to recordset
-- write to excel